website/react-magma-docs/src/pages/api/dropdown.mdx

```diff
@@ -427,14 +428,84 @@ export function Example() {
 }
 ```
 
-### Text only Expandable Menu Item
+## Expandable Menu Items Default Index
+
+The `defaultIndex` prop uses an array of zero based indices to enable an expandable group to be automatically opened.
+
+If the `isMulti` prop is set to `false`, the `defaultIndex` prop will take a single number, instead of an array.
```

Comment:
I think we have to be extra clear on where these props go because we are introducing a bunch of new components:

```suggestion
By default, the `isMulti` prop is true on `DropdownExpandableMenuGroup`, which means that the expandable dropdown will allow multiple items to be expanded at once. If you wish to limit the number of items open at once, you can set the `isMulti` prop to false. 

The `defaultIndex` prop on `DropdownExpandableMenuGroup` uses an array of zero based indices to enable expandable groups to be automatically opened.

When the `isMulti` prop is false, the `defaultIndex` prop will take a single number, instead of an array.
```